packages/mermaid/src/diagrams/packet/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { log } from '../../logger.js';
+
+export const styles = (options: any = {}) => {
```

Comment:
Maybe we should create `PacketStylesOptions` similar to `pie`?

https://github.com/mermaid-js/mermaid/blob/136f1c50e725b6303c1f0bebec828be0e604ed97/packages/mermaid/src/diagrams/pie/pieTypes.ts#L10-L35